modules/nvidia-gpu-aws-about-machines-and-machinesets.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
```

Comment:
I don't think we need this module since it's already in our docs:

https://docs.openshift.com/container-platform/4.11/machine_management/creating_machinesets/creating-machineset-aws.html

Also if we did need this content, it probably shouldn't go in the installation section since the GPU enablement on installation doesn't require the user to configure Machines/MachineSets. The content in the link about explains how to do that for AWS.

Reply:
REmoved this file from the Assembly